openreview/venue_request/venue_request.py

```diff
@@ -1617,6 +1617,12 @@ def setup_request_form(self):
                 'order': 47,
                 'required': False,
                 'hidden': True
+            },
+            'preferred_emails_groups': {
```

Comment:
Do the PCs have to contact us to ask for preferred email to be revealed or is there somewhere else where they can update `preferred_emails_groups`?

Reply:
They should contact us but we plan to enable it for large venues:  TMLR, ARR, CVPR, NeurIPS